winterfell/test/testClaimController.js

```diff
@@ -210,9 +211,40 @@ describe('SaveClaimController', function () {
       .expect(201, function() {
         Form.findOne({key: '1', user: targetUser._id}, function(error, doc) {
           should.not.exist(error);
+          should.exist(doc);
           doc.responses.should.deepEqual({key1: 'value1', key2: 'value2'});
           done();
         })
       });
   });
+
+  it('Should correctly calculate progress after save', function(done) {
```

Comment:
Oh I was not even thinking on that level, I was just having `testClaimController` is just testing all the endpoints in the `claimController.js` file. I think maybe we need to move the form logic into a different controller package if this one is becoming too cluttered.